scripts/src/metrics.ts

```diff
@@ -64,23 +64,24 @@ export function orderFailed(order: Order, relatedUser?: User) {
 	const unknownUser = { id: "no_id", appId: "no_id", appUserId: "no_id", walletAddress: "no_wallet" };
 
 	const error = order.error || unknownError;
-	const user = relatedUser || unknownUser;
-
-	const message = `
-## Order <${order.id}> transitioned to failed state:
-ID: <${order.id}> | Type: ${order.type} | Origin: ${order.origin}
-UserId: ${user.id} | AppId: <${user.appId}> | UserAppId: ${user.appUserId}
-Wallet: ${user.walletAddress}
-Error: ${safeString(error.message)} | Code: ${error.code}
+
+	order.forEachContext(context => {
+		const message = `
+## Order <${ order.id }> transitioned to failed state:
+ID: <${ order.id }> | Type: ${ context.type } | Origin: ${ order.origin }
+UserId: ${ context.user.id } | AppId: <${ context.user.appId }> | UserAppId: ${ context.user.appUserId }
+Wallet: ${ context.user.walletAddress }
+Error: ${ safeString(error.message) } | Code: ${ error.code }
 CreatedDate: ${order.createdDate.toISOString()} | LastDate: ${(order.currentStatusDate || order.createdDate).toISOString()}
 `;
-	const title = safeString(error.message);
-	statsd.event(title, message,
-		{ alert_type: "warning" },
-		{
-			order_type: order.type,
-			app_id: user.appId,
-			order_id: order.id,
-			order_origin: order.origin,
-			type: "failed_order" });
+		const title = safeString(error.message);
```

Comment:
No need to submit two events. Just one with this line doubled:
```
UserId: ${ context.user.id } | AppId: <${ context.user.appId }> | UserAppId: ${ context.user.appUserId }
```

Reply:
what about the `order_type`